worker/accounts.go

```diff
@@ -0,0 +1,130 @@
+package worker
+
+import (
+	"errors"
+	"sync"
+
+	"go.sia.tech/renterd/api"
+	"go.sia.tech/renterd/internal/consensus"
+	"go.sia.tech/renterd/rhp/v3"
+	"go.sia.tech/siad/types"
+	"golang.org/x/crypto/blake2b"
+)
+
+type (
+	// accounts stores the balance and other metrics of accounts that the
+	// worker maintains with a host.
+	accounts struct {
+		mu       sync.Mutex
+		accounts map[rhp.Account]*account
+
+		w *worker
```

Comment:
`nit` can you move this above the `mu` so we still have all fields covered by the mu below it

Reply:
Sure. I thought the rule is that the mutex applies to all fields below it in the same block. In case we ever want multiple in a single type.